pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -1280,8 +1280,8 @@ func TestFilterPluginsWithNominatedPods(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			registry := Registry{}
 			cfgPls := &config.Plugins{
-				PreFilter: &config.PluginSet{},
-				Filter:    &config.PluginSet{},
+				PreFilter: config.PluginSet{},
```

Comment:
you can remove these lines, that is the point :)

Reply:
cleaned all the missed ones.